### DISCUSSION TITLE
LeptonX Lite Blazor

### DISCUSSION THREAD

**Discussion by mehmetuken** [abpframework/abp#10227] [General] [answered]
Is there an estimated date for release?

**Comment by maliming** [comment, marked as answer]
It will be available with abp 5.0.

**Comment by recruitans** [comment]
and ?   @maliming

**Comment by maliming** [comment]
https://youtu.be/uLu2t5E8T-w?t=4442

<img width="630" alt="075ddfde2e361b3a51b4b981c028799" src="https://user-images.githubusercontent.com/6908465/146518086-3b8f32de-418b-436f-a1ce-aaf70aa25da7.png">

**Reply by Wiblox** [reply]
Can we follow the progress for blazor?

**Reply by Wiblox** [reply]
Hello @maliming  Is it possible to have an update  ?

**Reply by enisn** [reply]
Hi @Wiblox 

We are currently focused on working on LeptonX theme. We want to release it as soon as possible, too. Development of theme is almost finished but there are still unresolved issues on Theme. After resolving them, it'll be released very soon.

**Reply by BarisGurbuz** [reply]
I am also waiting for Lepton-X Lite Blazor

**Reply by Wiblox** [reply]
Thanks @enisn 
Do you think you can integrate the blazorise theme manager directly?

I found that on the current Lepton version it is more complicated to manage the whole theme.
Can I help you ?
After doing the cleaning I managed to use the blazorise theme manager.

Why not include it directly?

**Reply by enisn** [reply]
Hi @Wiblox 

We won't have Blazorise integration, It'll implemented on ABP Framework logic. We use blazorise to manage bootstrap dependency. We're not completely dependent on Blazorise.

We have plain angular implementation but don't have plain blazor or blazorise implementation. It'll be implemented for ABP Framework.

**Comment by hikalkan** [comment]
Register to https://kommunity.com/volosoft/events/abp-community-talks-20222-exploring-abps-new-user-interface-6cc09eaf to learn the latest news about the LeptonX project!

**Reply by Wiblox** [reply]
Thanks 👌

**Comment by karpine** [comment]
Is leptonx-lite available yet? cant see in 5.1 v.

**Reply by Wiblox** [reply]
https://docs.abp.io/en/abp/5.2/Road-Map

In 5.3 milestone for now.
